src/vm/codeman.h

```diff
@@ -140,6 +140,10 @@ typedef struct _hpCodeHdr
     PTR_EE_ILEXCEPTION  phdrJitEHInfo;
     PTR_BYTE            phdrJitGCInfo;
 
+#if defined(FEATURE_GDBJIT)
+    VOID*            pCalledMethods;
```

Comment:
This should be done by looking at the native code, or looking at the IL; not by recording this as an extra information at code generation time. A side-effect of doing it this way is that it will work for precompiled images.